superdesk/ws.py

```diff
@@ -54,10 +52,15 @@ def onMessage(self, payload, isBinary):
         self.factory.broadcast(payload, self.peer)
 
     def connectionLost(self, reason):
-        """Unregister on connection drop"""
+        """Unregister client on lost connection."""
         WebSocketServerProtocol.connectionLost(self, reason)
         self.factory.unregister(self)
 
+    def onClose(self, wasClean, code, reason):
+        """Unregister client on closed connection."""
+        WebSocketServerProtocol.onClose(self, wasClean, code, reason)
+        self.factory.unregister(self)
```

Comment:
i think those are only logging and if those would throw error the whole thing would stop anyhow